.travis.yml

```diff
@@ -18,10 +18,18 @@ matrix:
       env: TARGET=cs_dry_run
     - php: 5.5
       env: COMPOSER_FLAGS="--prefer-lowest"
+    # test 2.7 LTS
     - php: 5.6
       env: SYMFONY_VERSION=2.7.*
+    # test 2.8 LTS
     - php: 5.6
       env: SYMFONY_VERSION=2.8.*
+    # test the latest stable 3.x release
+    - php: 5.6
+      env: SYMFONY_VERSION=^3.0
```

Comment:
this does not make any sense. It should test the LTS

Reply:
Well, it's `^3` so since tomorrow the latest LTS and the version for this job will be the same!

Reply:
But this will use 3.3 which is not an LTS release.